README.md

```diff
@@ -177,7 +177,26 @@ Currently, the following clients are available for use:
     client enabled, age-restricted tracks are **not** guaranteed to play.
 
 ## Migration from Lavaplayer's built-in YouTube source
-This client is intended to be a drop-in replacement, however there are a couple of things to note:
+
+This client is intended as a direct replacement for Lavaplayer's `YoutubeAudioSourceManager`, which the maintainers
+deprecated in release [2.1.2](https://github.com/lavalink-devs/lavaplayer/releases/tag/2.1.2).
```

Comment:
I wouldn't include a hard reference to this specific deprecation as other forks may not necessarily have deprecated the source, or they might've deprecated it separately as is the case with my fork.